demo.sh

```diff
@@ -642,22 +575,25 @@ transfer_create() {
     _subtit "(sender) copying consignment to recipient data directory"
     _trace cp {"$send_data","$rcpt_data"}/"$CONSIGNMENT"
     # inspect consignment (output to file as it's very big)
-    _trace "${RGB[@]}" -d "$send_data" inspect \
-        "$send_data/$CONSIGNMENT" "$CONSIGNMENT.yaml"
-    _log "consignment exported to file: $CONSIGNMENT.yaml"
+    # TODO: Re-enable with `rgbx` tool
+    # _trace "${RGB[@]}" -d "$send_data" inspect \
+    #     "$send_data/$CONSIGNMENT" "$CONSIGNMENT.yaml"
+    # _log "consignment exported to file: $CONSIGNMENT.yaml"
```

Comment:
Inspection and other pro debug tasks has moved to a dedicated tool from `rgb-std` named `rgbx`. It will be good to install it alongside `rgb`, but this is too complex for my poor bash knowledge.

The reason is that `rgb` always has an internal runtime, which relies (and reads) on certain hardcoded directory structure, while `rgbx` allows to work with individual binary files without any runtime.